haiku-apps/nanodot/nanodot-1.1b.recipe

```diff
@@ -0,0 +1,48 @@
+SUMMARY="A minimal pattern-based MIDI sequencer"
+DESCRIPTION="nanodot is a rather minimal pattern-based MIDI \
+sequencer originally written for BeOS R3. Its main purpose in life was/is \
```

Comment:
Sorry, I just realized I forgot to give you a pointer to our wiki page about the recommended layout for the `DESCRIPTION` parameter: [HaikuPorter-Guidelines#description](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#description)

Our rules are quite strict about the word wrapping, so we really require that you change `DESCRIPTION` using the exact wrapping I had indicated in [this message](https://github.com/haikuports/haikuports/pull/2050#discussion_r159533057). Could you please make this small change?

I have successfully tested the app on both x86_gcc2 and x86_64.
This means we will be able to merge your PR once you have made the changes. Thanks!